pyannote/core/feature.py

```diff
@@ -237,7 +237,10 @@ def crop(
     def _repr_png_(self):
         from .notebook import repr_feature
 
-        return repr_feature(self)
+        try:
+            return repr_feature(self)
+        except ImportError:
+            return None
```

Comment:
Same. Would be nice to warn users about the existence of pyannote.core[notebook].